docs/literature_review.md

```diff
@@ -1,6 +1,87 @@
-Last update: 10 Feb 2020
+Last update: 12 Jan 2020
 
-Overview
+# Transformers review
+
+* Pre training on domain data from scratch is the best approach
+* A more computation efficient approach, if your dataset is small, is to create an expanded data from bringing the closest examples from the domain and pre training there
+* Training from scratch a smaller BERT performs much worse than distilling a bigger BERT into a smaller one
+
+## Don't stop pre training
+https://www.aclweb.org/anthology/2020.acl-main.740.pdf
+Gururangan et al, 2020
+
+![dapt and tapt](https://user-images.githubusercontent.com/4975761/104292786-ed369580-54c5-11eb-98c1-f67fef49ef0c.png)
+
+Domain adaptive pre training (DAPT), which is defined as pre trained on
+generic domain data like biomedical, is better than task adaptive pre training,
+which is defined as pre training on the same data distribution as the data of
+the task i.e. unlabelled examples, but their combination performs slightly
+better than just domain adaptive pre training.
+
+One of the reasons DAPT outperforms is that the are usually much more data
+in the domain than the task.
+
+![curated tapt](https://user-images.githubusercontent.com/4975761/104292923-0d665480-54c6-11eb-8062-1db6652097b3.png)
+
+You can approach DAPT performance by selecting a subset of the domain data using
```

Comment:
That's very interesting. Under which metric? Cosine similarity?

Reply:
Good point, i should have specified. Using some vectorisation like tfidf and then some fast way to calculate similarities